go.mod

```diff
@@ -52,3 +52,5 @@ require (
 	gopkg.in/yaml.v2 v2.4.0 // indirect
 	gopkg.in/yaml.v3 v3.0.1 // indirect
 )
+
+replace github.com/multiversx/mx-chain-vm-common-go v1.3.37-0.20230209123632-f0fbff86b763 => /home/bogdan/Elrond/elrond-vm-common
```

Comment:
Consider creating a branch on `mx-chain-vm-common-go` and using its commit hash instead.

Reply:
fixed